client/src/components/LeftDrawer/LeftDrawer.js

```diff
@@ -58,6 +62,18 @@ class LeftDrawer extends React.Component {
               </ListItem>
             </Link>
           ))}
+          <Link to={'/login'} key={4} className="link">

```

Comment:
You can remove **className="link"**, it was initially to remove the underline using CSS but it doesnt work on mac. You can replace it with:
**<Link to={'/login'} key={4} style={{ textDecoration: 'none' }}>**